var/spack/repos/builtin/packages/ncurses/package.py

```diff
@@ -7,6 +7,8 @@
 from glob import glob
 from os.path import exists, join
 from os import makedirs
+import re
+import os
```

Comment:
I think you can remove `re` now. You can also consolidate `os` if you want.

Reply:
Standardized the inputs for clarity.